src/analysis/retail/paladin/holy/modules/talents/BeaconOfVirtue.tsx

```diff
@@ -1,30 +1,59 @@
-import { TALENTS_PALADIN } from 'common/TALENTS';
 import CastEfficiencyBar from 'parser/ui/CastEfficiencyBar';
 import { GapHighlight } from 'parser/ui/CooldownBar';
 import Analyzer from 'parser/core/Analyzer';
-import talents from 'common/TALENTS/paladin';
+import TALENTS from 'common/TALENTS/paladin';
 import { explanationAndDataSubsection } from 'interface/guide/components/ExplanationRow';
 import SpellLink from 'interface/SpellLink';
 import { RoundedPanel } from 'interface/guide/components/GuideDivs';
 import { GUIDE_CORE_EXPLANATION_PERCENT } from '../../guide/Guide';
+import { getWordofGlorySpell } from 'analysis/retail/paladin/shared/constants';
+import SPELLS from 'common/SPELLS';
 
 class BeaconOfVirtue extends Analyzer {
   get guideSubsection(): JSX.Element {
     const explanation = (
       <p>
         <b>
-          <SpellLink spell={talents.BEACON_OF_VIRTUE_TALENT} />
+          <SpellLink spell={TALENTS.BEACON_OF_VIRTUE_TALENT} />
         </b>{' '}
-        is your best tool to handle burst AoE damage. It is quite mana expensive though, so you
-        should use it on cooldown only when healing is required.
+        is your best tool to handle burst AoE damage. It should be used nearly on cooldown, but due
+        its high mana cost, it's best to save it until your group needs significant healing. During{' '}
+        <SpellLink spell={TALENTS.BEACON_OF_VIRTUE_TALENT} />, you should always prioritize{' '}
+        <SpellLink spell={getWordofGlorySpell(this.selectedCombatant)} /> because it transfers more{' '}
+        healing to Beacons than <SpellLink spell={SPELLS.LIGHT_OF_DAWN_HEAL} /> does and ensure your
+        major cooldowns like <SpellLink spell={TALENTS.DIVINE_TOLL_TALENT} />{' '}
+        {this.selectedCombatant.hasTalent(TALENTS.DIVINE_FAVOR_TALENT) && (
+          <>
+            and <SpellLink spell={TALENTS.DIVINE_FAVOR_TALENT} />{' '}
+          </>
+        )}
+        are used in these windows.{' '}
+        {this.selectedCombatant.hasTalent(TALENTS.AVENGING_CRUSADER_TALENT) &&
+          this.selectedCombatant.hasTalent(TALENTS.HAMMER_AND_ANVIL_TALENT) &&
+          this.selectedCombatant.hasTalent(TALENTS.AWAKENING_TALENT) && (
+            <>
+              {' '}
+              As Lightsmith, when consuming an <SpellLink spell={TALENTS.AWAKENING_TALENT} /> proc,
+              you'll want to cast <SpellLink spell={TALENTS.BEACON_OF_VIRTUE_TALENT} /> prior to
+              casting the <SpellLink spell={SPELLS.JUDGMENT_CAST_HOLY} /> to make sure you get the
+              guaranteed <SpellLink spell={TALENTS.HAMMER_AND_ANVIL_TALENT} /> proc inside of it.
+            </>
+          )}
+        {this.selectedCombatant.hasTalent(TALENTS.LIGHTS_PROTECTION_TALENT) && (
+          <>
+            {' '}
+            Additionally, <SpellLink spell={TALENTS.LIGHTS_PROTECTION_TALENT} /> provides a 5%{' '}
+            damage reduction to your Beacon targets.
```

Comment:
You may want to put the 5 into a constant in case it ever gets changed by tuning or just get rid of the numeric specification and just say "damage reduction"